packages/framework/presence/src/notificationsManager.ts

```diff
@@ -147,17 +148,27 @@ class NotificationsManagerImpl<
 		broadcast: (name, ...args) => {
 			this.datastore.localUpdate(
 				this.key,
-				// @ts-expect-error TODO
-				{ rev: 0, timestamp: 0, value: { name, args: [...args] }, ignoreUnmonitored: true },
-				true,
+				{
+					rev: 0,
+					timestamp: 0,
+					value: { name, args: [...(args as JsonTypeWith<never>[])] },
```

Comment:
What does `JsonTypeWith` do?

Reply:
It is a type that represents values that roundtrip thru JSON. https://github.com/microsoft/FluidFramework/blob/d0dfd760093deb29bd95a29f5039b93013698866/packages/framework/presence/src/core-interfaces/jsonType.ts